src/Components/EnergyCalculator/Steps/Appliances.tsx

```diff
@@ -0,0 +1,126 @@
+import { FormattedMessage } from 'react-intl';
+import QuestionHeader from '../../QuestionComponents/QuestionHeader';
+import QuestionQuestion from '../../QuestionComponents/QuestionQuestion';
+import QuestionDescription from '../../QuestionComponents/QuestionDescription';
+import RHFOptionCardGroup from '../../RHFComponents/RHFOptionCardGroup';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { useParams } from 'react-router-dom';
+import { ReactComponent as WaterHeater } from '../Icons/WaterHeater.svg';
+import { ReactComponent as Heater } from '../Icons/Heat.svg';
+import { ReactComponent as Stove } from '../Icons/Stove.svg';
+import { ReactComponent as Washer } from '../Icons/Washer.svg';
+import PrevAndContinueButtons from '../../PrevAndContinueButtons/PrevAndContinueButtons';
+import { useDefaultBackNavigationFunction, useGoToNextStep } from '../../QuestionComponents/questionHooks';
+import { useContext } from 'react';
+import { Context } from '../../Wrapper/Wrapper';
+import useScreenApi from '../../../Assets/updateScreen';
+
+export const applianceStatusOptions = {
+  needsWaterHeater: {
+    icon: <WaterHeater className="option-card-icon" />,
+    text: {
+      props: {
+        id: 'applianceStatusOptions.needsWaterHeater',
+        default_message: 'Water Heater',
+      },
+    },
+  },
+  needsHvac: {
+    icon: <Heater className="option-card-icon" />,
+    text: {
+      props: {
+        id: 'applianceStatusOptions.needsHvac',
+        default_message: 'Heating, ventilation, and/or cooling',
+      },
+    },
+  },
+  needsStove: {
+    icon: <Stove className="option-card-icon" />,
+    text: {
+      props: {
+        id: 'applianceStatusOptions.needsStove',
+        default_message: 'Cooking stove/range',
+      },
+    },
+  },
+  needsDryer: {
+    icon: <Washer className="option-card-icon" />,
+    text: {
+      props: {
+        id: 'applianceStatusOptions.needsDryer',
+        default_message: 'Clothes dryer',
+      },
+    },
+  },
+};
+
+const Utilities = () => {
+  const { formData, setFormData } = useContext(Context);
```

Comment:
Same suggestion as the utilities pr.
```suggestion
  const { formData, setFormData } = useContext(Context);
  const energyDataAvailable = useEnergyFormData(formData);
```